reference/api/tasks.md

```diff
@@ -261,4 +272,65 @@ Get a single task.
 
 ## Cancel tasks
 
+<RouteHighlighter method="POST" route="/tasks/cancel?{task_uid}"/>
+
+Cancel an `enqueued` or `processing` task based on `uid`, `status`, `type`, `indexUid`, or date. Task cancelation is an atomic transaction, **either all tasks are successfully canceled or none are**.
+
+::: warning
+Using this route without any filters (POST `/tasks/cancel`) will result in the `missing_task_filters` error. This error prevents users from accidentally canceling all `enqueued` and `processing` tasks.
```

Comment:
Might be personal preference, but I would have gone the other way around with this note. That is, "to prevent users from accidentally cancelling all tasks, Meilisearch throws an error" instead of "the error prevents users from accidentally cancelling all tasks." Like, emphasise the reason why we don't want people to use the route without filters instead of the error itself.